doc/examples/config/config-storage/README.md

```diff
@@ -78,6 +78,18 @@ Since Tarantool version 3.2 keys could be added to the storage with the
 specified TTL. The keys guaranteed not to expire early, but they could live a
 bit longer than expected due to communications between replicas and master.
 
+If the schema used by the user is not upgraded, any request with specified TTL
+will end with an error. To check, if the schema needs an upgrade, use
+`box.internal.schema_needs_upgrade()`. To check if TTL requests are supported,
+use newly added field `config.storage.info.features.ttl`:
+```lua
+local info = conn.call('config.storage.info')
+if not info.features.ttl then
```

Comment:
It seems, we should suggest `if not (info.features or {}).ttl then` or `if not info.features or not info.features.ttl then` -- otherwise this code may fail on an attempt to index `nil` value.

Reply:
Updated.